modules/lang-painless/src/main/java/org/elasticsearch/painless/node/SSource.java

```diff
@@ -216,6 +220,19 @@ public void write() {
         visitor.visit(WriterConstants.CLASS_VERSION, classAccess, className, null, classBase, classInterfaces);
         visitor.visitSource(Location.computeSourceName(name, source), null);
 
+        // Write the a method to bootstrap def calls
```

Comment:
Doesn't look like it can be a non-static method given the order of the required arguments, and the reason for this method is explained below.  Thanks.